### PR TITLE
Exclude System.Net.Http from PortableFacades

### PR DESCRIPTION
**Pull request by genlu**
<details><summary>Ask Mode template not completed</summary>

### Customer scenario

What does the customer do to get into this situation, and why do we think this
is common enough to address for this release.  (Granted, sometimes this will be
obvious "Open project, VS crashes" but in general, I need to understand how
common a scenario is)

### Bugs this fixes

(either VSO or GitHub links)

### Workarounds, if any

Also, why we think they are insufficient for RC vs. RC2, RC3, or RTW

### Risk

This is generally a measure our how central the affected code is to adjacent
scenarios and thus how likely your fix is to destabilize a broader area of code

### Performance impact

(with a brief justification for that assessment (e.g. "Low perf impact because no extra allocations/no complexity changes" vs. "Low")

### Root cause analysis

How did we miss it?  What tests are we adding to guard against it in the future?

### How was the bug found?

(E.g. customer reported it vs. ad hoc testing)

### Test documentation updated?

If this is a new non-compiler feature or a significant improvement to an existing feature, update https://github.com/dotnet/roslyn/wiki/Manual-Testing once you know which release it is targeting.

</details>

### CODE REVIEW

**Comment by genlu** [comment]
FYI @jasonmalinowski

File: src/Setup/DevDivVsix/PortableFacades/PortableFacades.swr [hunk 1]
```diff
@@ -18,7 +18,6 @@ folder InstallDir:\Common7\IDE\PrivateAssemblies
     file source="$(NuGetPackageRoot)\System.IO.FileSystem.Primitives\4.3.0\lib\net46\System.IO.FileSystem.Primitives.dll" vs.file.ngen=yes
     file source="$(NuGetPackageRoot)\System.IO.Pipes.AccessControl\4.3.0\runtimes\win\lib\net46\System.IO.Pipes.AccessControl.dll" vs.file.ngen=yes
     file source="$(NuGetPackageRoot)\System.IO.Pipes\4.3.0\runtimes\win\lib\net46\System.IO.Pipes.dll" vs.file.ngen=yes
-    file source="$(NuGetPackageRoot)\System.Net.Http\4.3.0\runtimes\win\lib\net46\System.Net.Http.dll" vs.file.ngen=yes
```
Comment (jaredpar) [inline src/Setup/DevDivVsix/PortableFacades/PortableFacades.swr]: Should remove System.Diagnostics.DiagnosticSource as well. It's only needed by System.Net.Http

Comment (jasonmalinowski) [reply src/Setup/DevDivVsix/PortableFacades/PortableFacades.swr]: Ah, hmm, which then rolls back 2705fd4d9071b8e2ed15662a2e345af875caae1b?

Comment (jasonmalinowski) [reply src/Setup/DevDivVsix/PortableFacades/PortableFacades.swr]: Makes sense. @genlu want to take care of it?

Comment (jaredpar) [reply src/Setup/DevDivVsix/PortableFacades/PortableFacades.swr]: That's at least my understanding last time I looked through all the nuspec files and got the dependency order.

CC @terrajobst as yet another reminder how impossible this is for us to deal with.

Comment (genlu) [reply src/Setup/DevDivVsix/PortableFacades/PortableFacades.swr]: Done

**Pull request merged**